crates/bevy_ecs/src/archetype.rs

```diff
@@ -44,9 +44,9 @@ pub struct AddBundle {
 
 #[derive(Default)]
 pub struct Edges {
```

Comment:
Why is this type `pub` at all?

Reply:
It could be useful to figure out the Archetype/Bundle graph. Probably useful for diagnostics and gameplay design optimization if users wanted to analyze the growth/use of the graph as their games progress.

I agree it otherwise is sort of an implementation detail.